graphs.py

```diff
@@ -50,19 +52,43 @@ def render_graph(graph_data: dict):
 
     min_followers = min(graph_data['info_followers_per_day'])
     max_followers = max(graph_data['info_followers_per_day'])
-    print(max_followers)
+
     min_followers = floor(min_followers / 100) * 100  # round to bottom 100
     max_followers = ceil(max_followers / 100) * 100  # round to upper 100
-    max_tweets_likes = max(max(graph_data['info_tweets_per_day'], graph_data['info_likes_number']))
-    # (max(graph_data['info_tweets_per_day'], graph_data['info_likes_number'])  -> returns a list
-    max_tweets_likes = ceil(max_tweets_likes / 10) * 10 # round to upper 10
-    print(max_tweets_likes)
+    max_tweets_likes = max(graph_data['info_tweets_per_day'] + graph_data['info_likes_number'])
```

Comment:
This was wrong before. You had there two lists, and you took the "bigger" (whatever that means), and then the second `max()` took the biggest element from that list, which is wrong.
Example:
```
>>> max([1,2], [0,3])
[1, 2]
```

so in the end you'd end with value of `2` as the biggest number, which is obviously wrong.